README.md

```diff
@@ -223,6 +223,7 @@ With this `{rootDir}/src/ui/tsconfig.json`:
   - `endpoint` (`string`) - cdn host to upload files
   - `compress` (`boolean`) - upload also gzip and brotli compressed versions of files
   - `additionalPattern` (`string[]`) — patterns for uploading additional files. By default, only files generated by webpack are loaded.
+- `sentryConfig` (`Options`) — `@sentry/webpack-plugin` [configuration options](https://www.npmjs.com/package/@sentry/webpack-plugin). Requires `hiddenSourceMap: false` and `disableSourceMapGeneration: false`.
```

Comment:
There is no documentation
<img width="837" alt="image" src="https://github.com/gravity-ui/app-builder/assets/10867286/2f89d6c1-9d64-432c-b761-1470eaaffa0b">

Reply:
It opens from time to time, I can replace it with https://github.com/getsentry/sentry-javascript-bundler-plugins/blob/main/packages/webpack-plugin/README_TEMPLATE.md#example